src/domain/airplane/repositories/airplane.repository.ts

```diff
@@ -3,20 +3,20 @@ import PlaneId from "../value-objects/plane-id";
 
 interface AirplaneRepository
 {
-    // Save an airplane aggregate
-  save(airplane: Airplane): Promise<void>;
+  // Save an airplane aggregate
+  save(airplane: Airplane): any;
 
   // Find an airplane by its ID
-  findById(planeId: PlaneId): Promise<Airplane | null>;
+  // findById(planeId: PlaneId): Promise<Airplane | null>;
 
-  // Find all airplanes
-  findAll(): Promise<Airplane[]>;
+  // // Find all airplanes
+  // findAll(): Promise<Airplane[]>;
```

Comment:
you can uncomment these now. It's generally not a good idea to check in commented out codes